src/components/UserSelector.tsx

```diff
@@ -12,24 +28,36 @@ export const UserSelector: React.FC = () => {
           className="button"
           aria-haspopup="true"
           aria-controls="dropdown-menu"
+          onClick={handleMenu}
         >
-          <span>Choose a user</span>
+          {user ? (
+            <span>{user.name}</span>
+          ) : (
+            <span>Choose a user</span>
+          )}
```

Comment:
```suggestion
          {user 
            ? (<span>{user.name}</span> ) 
            : (<span>Choose a user</span>)
          }
```

Reply:
Hi, about cleaning the form - in the task it is written "keep the name and email after the successful submit but clear a comment text;". About email in black line box - in the task, the layout of the component PostDetails
 was ready and it showed the user's name, not his email.
![gh](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/91830857/c9eeed22-87e1-4c3c-be70-0d7a8e66b105)